assets/src/edit-story/components/panels/font.js

```diff
@@ -106,39 +129,79 @@ function FontPanel({ selectedElements, onSetProperties }) {
       title={__('Font', 'web-stories')}
       onSubmit={handleSubmit}
     >
-      {fonts && (
+      <Row>
+        {fonts && (
+          <SelectMenu
+            ariaLabel={__('Font family', 'web-stories')}
+            options={fonts}
+            value={state.fontFamily}
+            isMultiple={fontFamily === ''}
+            onChange={(value) => {
+              const currentFontWeights = getFontWeight(value);
+              const currentFontFallback = getFontFallback(value);
+              const fontWeightsArr = currentFontWeights.map(
+                ({ thisValue }) => thisValue
+              );
+              const newFontWeight =
+                fontWeightsArr && fontWeightsArr.includes(state.fontWeight)
+                  ? state.fontWeight
+                  : 400;
+              setState({
+                ...state,
+                fontFamily: value,
+                fontWeight: parseInt(newFontWeight),
+                fontWeights: currentFontWeights,
+                fontFallback: currentFontFallback,
+              });
+            }}
+          />
+        )}
+      </Row>
+      <Row>
         <SelectMenu
-          label={__('Font family', 'web-stories')}
-          options={fonts}
-          value={state.fontFamily}
-          isMultiple={fontFamily === ''}
-          onChange={(value) => {
-            const currentFontWeights = getFontWeight(value);
-            const currentFontFallback = getFontFallback(value);
-            const fontWeightsArr = currentFontWeights.map(
-              ({ thisValue }) => thisValue
-            );
-            const newFontWeight =
-              fontWeightsArr && fontWeightsArr.includes(state.fontWeight)
-                ? state.fontWeight
-                : 400;
-            setState({
-              ...state,
-              fontFamily: value,
-              fontWeight: parseInt(newFontWeight),
-              fontWeights: currentFontWeights,
-              fontFallback: currentFontFallback,
-            });
-          }}
+          ariaLabel={__('Font style', 'web-stories')}
```

Comment:
Note: This select is most likely for `fontWeight` as the removed code above -- `fontStyle` is resolved via a Toggle:
<img width="378" alt="Screenshot 2020-03-03 at 09 02 08" src="https://user-images.githubusercontent.com/3294597/75754820-ae747280-5d2d-11ea-9a05-6101b60a1ba4.png">

Reply:
Panel added to textStyle panel. I think font panel is not necessary since we have a textStyle panel.